ckanext/who_afro/assets/css/userprofile.css

```
@@ -0,0 +1,345 @@
+    .dashboard-organisation-main h2.page-heading.mb-4{
+        text-align: left;
+    }
+}
+
+@media only screen and (min-width: 992px) {
+    .profile-left-info-details {
+      justify-content: flex-end;
+    }
+    .profile-left-info-details li:not(:first-child)::before {
+      content: '|';
+      color: var(--who-blue-65-percent);
+      margin-left: 16px;
+      margin-right: 16px;
+    }
+    .upper-content ul {
+      list-style-type: none;
+      padding: 0;
+      display: flex;
+      color: var(--primary-navy);
+        font-size: 16px;
+        font-weight: 400;
+        line-height: 24px;
+      margin-bottom: 0;
+    }
+
+    .right-content ul {
+        justify-content: flex-end;
+    }
+}
+
+.right-content .sysadmin-item a {
+    padding-right: 0;
+    text-decoration: underline;
+}
+.right-content ul li:nth-child(1) {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  margin-right: 19px;
+}
+.right-content ul li a {
+  font-weight: 700;
+}
+.right-content ul li:nth-child(2) {
+  border: var(--primary-navy) thin solid;
+  border-radius: 4px;
+  padding: 12px 16px;
+  cursor: pointer;
+}
+
+.right-content .logout-btn {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.right-content .logout-btn:hover {
+    text-decoration: none;
+    background: var(--who-blue-65-percent);
+}
+
+.right-content .logout-btn a {
+    text-decoration: none;
+}
+
+.profile-left-info-details li a {
+    text-decoration: underline;
+}
+
+/* tabs */
+.secondary {
+    content: none;
+    display: none;
+}
+
+.wrapper .primary {
+    width: 100%;
+    border: 0;
+}
+
+.wrapper:before {
+    border-right: 0;
+}
+
+.wrapper {
+    box-shadow: none;
+    border: 0;
+}
+
+.page-header {
+    background-color: white;
+}
+
+div.module-content {
+    border: var(--who-blue-65-percent) thin solid;
+    border-top: none;
+}
+
+header,
+header ul.nav.nav-tabs {
+    width: calc(100% + 1px);
+}
+
+header ul.nav.nav-tabs li:first-child {
+    width: 25%;
+}
+
+header ul.nav.nav-tabs li:not(:first-child) {
+    width: 24.3%;
+    margin-left: 0.7%;
+}
+
+header ul.nav.nav-tabs li:last-child {
+    width: 24.3%;
+    margin-left: 0.7%;
+}
+
+.page-header .nav-tabs li.active a, .page-header .nav-tabs a:hover {
+    border: var(--who-blue-65-percent) thin solid;
+    border-bottom-color: transparent;
+}
+
+.nav-tabs .nav-link, .page-header .nav-tabs li a,
+.nav-tabs .nav-link, .page-header .nav-tabs li a:hover {
+    background-color: var(--who-blue-ten-percent);
+}
+
+.nav-tabs,
+.nav-tabs .nav-link, .page-header .nav-tabs li a {
+    border-bottom-color: var(--who-blue-65-percent);
+}
+
+.nav.nav-tabs li a {
+    color: var(--primary-navy);
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 20px;
+    text-align: center;
+}
+
+@media only screen and (max-width: 510px) {
+    header ul.nav.nav-tabs li,
+    header ul.nav.nav-tabs li:first-child,
+    header ul.nav.nav-tabs li:not(:first-child),
+    header ul.nav.nav-tabs li:last-child {
```

This looks a little odd to me: the set seems to add up to just be all children?
```suggestion
    header ul.nav.nav-tabs li > * {
```

---

As suggested I have merged the classes into one : 
```
    .user-profile-page header ul.nav.nav-tabs li {
        width: 100% !important;
    }
```